packages/core/src/interpreter.ts

```diff
@@ -365,7 +373,8 @@ export class Interpreter<
   /**
    * Stops the interpreter and unsubscribe all listeners.
    */
-  public stop(): this {
+  public _stop(): this {
+    evict(this, 'initial');
```

Comment:
As far as I was able to analyze this, this is redundant and basically just a noop because we never `memo(actor, 'initial', ...)` 
```suggestion
```